docs/CONTRIBUTING.md

```diff
@@ -112,6 +117,43 @@ In our GitHub Actions CI, we use CentOS 7 for maximum compatibility.
 If you need your builds to support older distros, run the build commands
 inside a Docker container with all the build requirements installed.
 
+## Testing
+
+There are three kinds of tests in code-server:
+
+1. unit tests
+2. integration tests
+3. end-to-end tests
+
+### Unit Tests
+
+Our unit tests are written in TypeScript and run using [Jest](https://jestjs.io/), which is a JavaScript testing framework.
```

Comment:
small note: I'd suggest phrasing as `[Jest testing framework]` because it makes the link bigger, and hence more clickable when you're on mobile. It can be really tough to click small links with your thumb on a small display! Links probably need some minimum length for accessibility purposes I think

Reply:
really really good point! i'll make that change now!